lib/index.js

```diff
@@ -0,0 +1,169 @@
+require("dotenv").config();
+const { OpenAI } = require("openai");
+const fs = require("fs");
+const axios = require("axios");
+const cheerio = require("cheerio");
+const pretty = require("pretty");
+
+const OUTPUT_FOLDER = "summaries";
```

Comment:
Keeping a global variable to store the news summaries that our AI agent will do for us.